bob/src/hw_metrics_collector.rs

```diff
@@ -91,10 +93,10 @@ impl HWMetricsCollector {
             match cpu_s_c.iowait() {
                 Ok(iowait) => {
                     gauge!(CPU_IOWAIT, iowait);
-                },
+                }
```

Comment:
Most of the time formatting changes leads to a merge conflicts in the future. Right now you can keep them, but in the future requests, please, do not commit formatting changes in the code, that is not directly affected by the changes, required by issue itself